powertools-tracing/pom.xml

```diff
@@ -115,6 +119,13 @@
             <artifactId>assertj-core</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
FYI:
in `main` (v1) we are using log4j everywhere, in `v2` we are switching to slf4j. I would say it would be nice to keep this with a log4j in memory appender or so for the test, but seeing as we would then pull it immediately out and replace it with slf4j based stuff anyway and we're in a test, it seems a bit silly 👍

Reply:
+1000

Reply:
can we remove this? Please use the log4j-slf4j-impl that we have elsewhere...